### PR TITLE
Tweak a few docstrings.

### PR DESCRIPTION
**Pull request by anntzer**
## PR checklist

- [ ] "closes #0000" is in the body of the PR description to [link the related issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue)
- [ ] new and changed code is [tested](https://matplotlib.org/devdocs/devel/testing.html)
- [ ] *Plotting related* features are demonstrated in an [example](https://matplotlib.org/devdocs/devel/document.html#write-examples-and-tutorials)
- [ ] *New Features* and *API Changes* are noted with a [directive and release note](https://matplotlib.org/devdocs/devel/coding_guide.html#new-features-and-api-changes)
- [ ] Documentation complies with [general](https://matplotlib.org/devdocs/devel/document.html#write-rest-pages) and [docstring](https://matplotlib.org/devdocs/devel/document.html#write-docstrings) guidelines

### CODE REVIEW

File: lib/matplotlib/colors.py [hunk 1]
```diff
@@ -1245,21 +1245,19 @@ def __init__(self, vmin=None, vmax=None, clip=False):
...
-            values with specific colors for over or under.
-
-            If *clip* is ``True``, values outside ``[vmin, vmax]`` are set to 0 or 1,
-            depending on which boundary they're closer to. This makes these values
-            indistinguishable from regular boundary values and can lead to
-            misinterpretation of the data.
+            If clipping is off, values outside the range ``[vmin, vmax]`` are
+            also transformed, resulting in values outside ``[0, 1]``.  This
+            behavior is usually desirable, as colormaps can mark these *over*
+            and *under* values with specific colors.
```
Comment (QuLogic) [inline lib/matplotlib/colors.py:1255]: Should be same order as `vmin, vmax` or `0, 1`:
```suggestion
            behavior is usually desirable, as colormaps can mark these *under*
            and *over* values with specific colors.
```

File: lib/matplotlib/colors.py [hunk 2]
```diff
@@ -1567,15 +1565,15 @@ def __init__(self, vcenter=0, halfrange=None, clip=False):
...
             ``[vmin, vmax]``.
 
-            If clipping is off, values outside the range ``[vmin, vmax]`` are also
-            transformed, resulting in values outside ``[0, 1]``. For a
-            standard use with colormaps, this behavior is desired because colormaps
-            mark these outside values with specific colors for *over* or *under*.
+            If clipping is off, values outside the range ``[vmin, vmax]`` are
+            also transformed, resulting in values outside ``[0, 1]``.  This
+            behavior is usually desirable, as colormaps can mark these *over*
+            and *under* values with specific colors.
```
Comment (QuLogic) [inline lib/matplotlib/colors.py:1571]:
```suggestion
            behavior is usually desirable, as colormaps can mark these *under*
            and *over* values with specific colors.
```

File: lib/matplotlib/colors.py [hunk 3]
```diff
@@ -1852,14 +1850,13 @@ def forward(values: array-like) -> array-like
...
-        standard use with colormaps, this behavior is desired because colormaps
-        mark these outside values with specific colors for *over* or *under*.
-
-        If ``True`` values falling outside the range ``[vmin, vmax]``,
-        are mapped to 0 or 1, whichever is closer. This makes these values
-        indistinguishable from regular boundary values and can lead to
-        misinterpretation of the data.
+        transformed by the function, resulting in values outside ``[0, 1]``.
+        This behavior is usually desirable, as colormaps can mark these *over*
+        and *under* values with specific colors.
```
Comment (QuLogic) [inline lib/matplotlib/colors.py:1855]:
```suggestion
        This behavior is usually desirable, as colormaps can mark these *under*
        and *over* values with specific colors.
```

File: lib/matplotlib/colors.py [hunk 4]
```diff
@@ -1957,14 +1954,13 @@ class PowerNorm(Normalize):
...
-        a standard use with colormaps, this behavior is desired because colormaps
-        mark these outside values with specific colors for *over* or *under*.
-
-        If ``True`` values falling outside the range ``[vmin, vmax]``,
-        are mapped to 0 or 1, whichever is closer. This makes these values
-        indistinguishable from regular boundary values and can lead to
-        misinterpretation of the data.
+        transformed by the power function, resulting in values outside ``[0, 1]``.
+        This behavior is usually desirable, as colormaps can mark these *over*
+        and *under* values with specific colors.
```
Comment (QuLogic) [inline lib/matplotlib/colors.py:1959]:
```suggestion
        This behavior is usually desirable, as colormaps can mark these *under*
        and *over* values with specific colors.
```

File: lib/matplotlib/colors.py (hunk 1 above)
Comment (anntzer) [reply lib/matplotlib/colors.py]: Thanks, fixed (throughout).

**Review by QuLogic — APPROVED** [review]

**Comment by QuLogic** [comment]
Not going to bother with a backport.

**Pull request merged**